pkg/apimachinery/utils/meta.go

```diff
@@ -46,6 +46,19 @@ const AnnoKeyRepoPath = "grafana.app/repoPath"
 const AnnoKeyRepoHash = "grafana.app/repoHash"
 const AnnoKeyRepoTimestamp = "grafana.app/repoTimestamp"
 
+// Annotations used to store manager properties
+
+const AnnoKeyManagerKind = "grafana.app/managedBy"
+const AnnoKeyManagerIdentity = "grafana.app/managerId"
+const AnnoKeyManagerAllowsEdits = "grafana.app/managerAllowsEdits"
+const AnnoKeyManagerSuspended = "grafana.app/managerSuspended"
```

Comment:
```suggestion
// AnnoKeyManagerSuspended indicates ????
const AnnoKeyManagerSuspended = "grafana.app/managerSuspended"
```

I have no idea what this means :)

Reply:
If set to true, this will indicate to the manager that this resource has been temporarily excluded from being managed and should be ignored.

This is a good catch, let me update the doc comments on the manager / source properties.

Reply:
Updated.